### PR TITLE
Testing tic tac toe - JavaScript

### PR DESCRIPTION
**Pull request by gitnyasha**
- the testing game logic of the game with jest

### CODE REVIEW

File: src/playerController.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    }
+  }
+
+  return {
+    addPlayer(name, symbol) {
+      return new Player(name, symbol);
+    },
+    addMove(move) {
+      this.moves.push(move);
+    },
```
Comment (maelfosso) [inline src/playerController.js:18]: There is no test concerning this method

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello @gitnyasha @konyan 

1. Please, you must give more details in your PR

2. There are tests that should be written
**gameBoardController**: 
- `resetBoard()` 
- `checkWinningMove()` : The test you wrote is not bad but we should see interaction between players and you should show us diagonal and vertical moves
- `hidePlayerForm()`: Show us that at the beginning of the game, the players form is hidden
- `showWinner()`: Join that test with the `checkWinningMove()` one. 

2. Read the comments below

Kindly make the changes and submit a new code review request
Thanks

Mael FOSSO

**Comment by bellom** [comment]
> * the testing game logic of the game with jest
This 👇 is an example of what @maelfosso is saying on your pull request summary
image

**Review by bellom — APPROVED** [review]
## Hi @gitnyasha  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**